mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -155,29 +155,29 @@ def _extract_run_id(artifact_uri):
         artifact_path = extract_and_normalize_path(artifact_uri)
         return artifact_path.split("/")[3]
 
-    def _call_endpoint(self, service, api, json_body=None, path_params=None):
+    def _call_endpoint(self, service, api, params=None):
         """
         Calls the specified REST endpoint with the specified JSON body and path parameters.
 
         Args:
             service: The service to call.
             api: The API to call.
-            json_body: The JSON body of the request.
-            path_params: The path parameters to substitute into the endpoint URI.
+            params: The request parameters.
```

Comment:
Can we be more specific about what `params` means?

Reply:
updated!